src/Commands/CommandBus.php

```diff
@@ -204,12 +204,12 @@ protected function handler(Update $update): Update
     {
         $message = $update->getMessage();
 
-        if ($message !== null && $message->has('text') && $message->has('entities')) {
-
+        if ($message->has('entities')) {
```

Comment:
Yeah, I wanted to know your thoughts on the `->getMessage()` method.

At the moment it returns a "message (collection)" or "edited_message (collection)" or null.

I think it might be appropriate to return an EMPTY collection() rather than null....but I wanted to discuss. In the mean time I've reverted to checking for null. Will push soon.